website/cue/reference/components/sinks/aws_kinesis_streams.cue

```diff
@@ -40,11 +40,6 @@ components: sinks: aws_kinesis_streams: components._aws & {
 			proxy: enabled: true
 			request: {
 				enabled:                    true
-				concurrency:                5
-				rate_limit_duration_secs:   1
-				rate_limit_num:             5
-				retry_initial_backoff_secs: 1
-				retry_max_duration_secs:    10
 				timeout_secs:               30
```

Comment:
I am curious if this 30 was intentional. It looks like it used to be 10 and then was bumped to 30 in #439 with no explicit rationale. I'd be game to just dump this one and the Firehose one and put them to their default.

Reply:
I'm game to shift to 60s absent any comment explaining why the value isn't arbitrary, agreed.

Reply:
Resolved in f2734ac